registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -64,6 +64,39 @@
         }
       ]
     },
+    "0x4af2427da3de2ce21d62560090698735f50ccee680b09941a1e21335993c5f9a": {
+      "tokenId": "0x4af2427da3de2ce21d62560090698735f50ccee680b09941a1e21335993c5f9a",
+      "deployer": "0x2347bA6939476B0BE50f4E8Ba7C2419555555225",
+      "originalMinter": null,
+      "prettySymbol": "MSS",
+      "decimals": 18,
+      "deploySalt": "0x158396dcfbc0792392d26a43c6e2723c4e2232783d2ba06bbc551bcd42a89730",
+      "originAxelarChainId": "binance",
+      "tokenType": "customInterchain",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/mss.svg"
+      },
+      "coinGeckoId": "mintstakeshare",
+      "deploymentMessageId": "",
+      "chains": [
+        {
+          "symbol": "MSS",
+          "name": "MintStakeShare",
+          "axelarChainId": "binance",
+          "tokenAddress": "0xf0F14CbD7ce6753BC209EB0d8F67fC84cccB9B2F",
+          "tokenManager": "0xB9d897478bf9adeb1e74E8a5BF3596786A25a6F0",
+          "tokenManagerType": "mintBurnFrom"
```

Comment:
Any color on why this was deployed as `mintBurnFrom` instead of `mintBurn`? cc @benjamin852